application/run_debug.bat

```diff
@@ -1,4 +1,4 @@
 set environment=development
 set FLASK_APP=main.py
-set FLASK_DEBUG=1
+set FLASK_DEBUG=0
```

Comment:
I'd rather not commit this, maybe make it a new .bat file `run_no_debug.bat` or sth.